packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -680,11 +684,8 @@ The `VersionQuery` construct encapsulates a version of Deadline and the location
 
 You can specify a Deadline version as follows:
 ```ts
-const version = VersionQuery.exact(stack, 'ExactVersion', {
-  majorVersion: '1',
-  minorVersion: '2',
-  releaseVersion: '3',
-  patchVersion: '4'
+const version = new VersionQuery(stack, 'ExactVersion', {
```

Comment:
```suggestion
const version = new VersionQuery(stack, 'Version', {
```

super minor, but I wonder if we should change the construct ID to just `Version` since we've removed the `exact` method. What do you think?